backend/src/main/java/com/praksa/auction/controller/PersonController.java

```diff
@@ -50,11 +57,18 @@ public ResponseEntity<?> logIn(@Valid @RequestBody LogInDto loginInfo){
         String jwt = jwtUtils.generateJwtToken(authentication);
         PersonDetails userDetails = (PersonDetails) authentication.getPrincipal();
         BasicUserInfoDto basicPersonInfo = new BasicUserInfoDto(userDetails.getId(), userDetails.getFirstName(), userDetails.getLastName(), userDetails.getEmail());
-        return ResponseEntity.ok(new JwtResponseDto(jwt,basicPersonInfo));
+        return ResponseEntity.ok(new JwtResponseDto(jwt, basicPersonInfo));
     }
 
     @PostMapping("/register")
     public ResponseEntity<?> createAccount(@Valid @RequestBody RegistrationDto signUpRequest) {
+
+        String validation = signUpRequest.validateRegistration();
```

Comment:
Again logic in controller, `personService` should handle logic for validation. You should pass `signUpRequest` to the `createAccount` method